status-im/status-legacy
# [#9682] Rename the Dapps tab to Browser

fixes #9682 

### Summary

• Renamed the tab bar text label to 'Browser'
• Swapped the tab bar icon to browser
• Overwritten existing browser icon with a slightly refreshed one to better match the appearance of other icons in the tab bar

### Review notes
• baby's first PR, pls correct if necessary :) 

### Testing notes
• test if the change made matches this 

<img width="1175" alt="Screenshot 2019-12-17 at 09 39 11" src="https://user-images.githubusercontent.com/4932774/71067828-8e450680-2175-11ea-9534-8327ab78649e.png">

#### Platforms

- Android
- iOS
#### Areas that maybe impacted
bottom Tab Bar

### Steps to test


- Open Status
- Verify the bottom tab bar's appearance



status: ready
